pootle/settings/40-apps.conf

```diff
@@ -123,6 +123,9 @@ ACCOUNT_LOGIN_ON_EMAIL_CONFIRMATION = True
 # Automatically sign users in upon password reset completion
 ACCOUNT_LOGIN_ON_PASSWORD_RESET = True
 
+# True to always remember user session. False to not remember. None to ask user.
+ACOUNT_SESSION_REMEMBER = True
```

Comment:
Is there a reason we aren't setting this to None? Would that require a UI change?  Just seems like the much more obvious way for Pootle the product and I'm not seeing why True was the one chosen.